app/main.ts

```diff
@@ -1,11 +1,14 @@
 import { app, BrowserWindow, ipcMain, screen } from "electron";
 import * as path from "path";
 import * as url from "url";
+import * as fs from "fs-extra";
 
 // Initialize remote module
 require("@electron/remote/main").initialize();
 
 let win: BrowserWindow = null;
+const userDirectory =
```

Comment:
Move this whole variable into a separate file, call it electron-config.ts, import the variable here then

Reply:
done